routers/web/repo/view.go

```diff
@@ -696,8 +696,8 @@ func checkCitationFile(ctx *context.Context, entry *git.TreeEntry) {
 	}
 }
 
-// Home render repository home page
-func Home(ctx *context.Context) {
+// AddHeaderCtxData adds context data for the repo header
+func AddHeaderCtxData(ctx *context.Context) {
```

Comment:
this not only adds the feed header it also respond with the actual feed if isFeed is true

Reply:
I'd like to say this name is quite unclear. `AddHeaderCtxData`: what kind of `data` .....

And it should be exposed as a public function.